tasks/islandora.yml

```diff
@@ -133,3 +133,24 @@
 - name: Run repo.enable.sh
   command: >
     /opt/php/bin/drush -y -u 1 -r /srv/{{ islandora_site }}/drupal php-script /opt/oulib/island/bin/repo_enable.php
+
+- name: Modify the script of oEmbed module
+  template:
+    src: "oembedcore.module.j2"
+    dest: "/srv/{{islandora_site}}/drupal/sites/all/modules/oembed/oembedcore.module"
+    owner: "apache"
+    group: "apache"
+
+- name: Modify the script of subpathauto module
+  template:
+    src: "subpathauto.module.j2"
+    dest: "/srv/{{islandora_site}}/drupal/sites/all/modules/subpathauto/subpathauto.module"
+    owner: "apache"
+    group: "apache"
+
+- name: Modify the djatoka url of Islandora bookreader module
```

Comment:
Since we've forked this module, maybe we could just make this change in our version?

We might also want to fork oembed and subpathauto and handle fixes in the same way to keep security updates to those modules from overwriting our changes.